### PR TITLE
Added a sidebar tab to view untagged notes.

### PR DESCRIPTION
**Pull request by SuhanKoh**
See: #464 
I have to modify setProperty(TAGS_PROPERTY, "") because when the note created, the tags are null, but when you remove a tag, the tags is actually empty string. Based on my limited understanding, I cannot do an or query.

screenshot_20180426-212506

### CODE REVIEW

**Comment by roundhill** [comment]
Since this will only be useful to a small amount of users, I think it'd be better to put this in a footer row of the tags list (if you have untagged notes) instead of a header item. Is that possible?

**Comment by SuhanKoh** [comment]
Above the "Setting" text? Or before the divider line?

**Comment by roundhill** [comment]
@drw158 do you have any thoughts on what the UX should be here? We'd need to apply this to all of the apps.

**Comment by drw158** [comment]
Because of it's use, it should be within the tags list. I think it should be at the bottom of the tags list — not pinned or "sticky". So, in this case the button would sit directly below the "testing" tag.

Some design work needs to go into making it look better though. Because it would be in the tags list, it'll look strange with the icon. Although, you want it to stand out and not look like a regular tag.

**Comment by SuhanKoh** [comment]
Can you provide a simple mock up?

**Comment by drw158** [comment]
artboard

or if you want to add an icon

screen shot 2018-05-23 at 9 28 01 am

**Comment by SuhanKoh** [comment]
Updated UI.

File: Simplenote/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -43,6 +43,7 @@
     <string name="search">Search notes</string>
     <string name="notes">All Notes</string>
     <string name="trash">Trash</string>
+    <string name="untaggedNotes">Untagged Notes</string>
```
Comment (roundhill) [inline Simplenote/src/main/res/values/strings.xml]: Can you please make this `untagged_notes` to match the style of the other string names?

**Comment by SuhanKoh** [comment]
- [ ] Remove top and bottom margin
- [ ] Fix the string id to match existing style

**Comment by SuhanKoh** [comment]
Screenshot on removed margin.
ss

**Comment by roundhill** [comment]
Looks/works great! Thanks for the contribution @SuhanKoh!

**Comment by alialwadie100** [comment]
make php function withe my email alialwadie@outlook.com

**Pull request merged**